integration/Dockerfile

```diff
@@ -16,4 +16,4 @@ RUN mv ./kubectl /usr/local/bin
 
 COPY bin/kubectl-gadget /bin/kubectl-gadget
 COPY --from=build_base /build/gadget-integration.test /bin/gadget-integration.test
-CMD ["/bin/sh", "-c", "gadget-integration.test -integration -test.timeout 20m"]
+CMD ["/bin/sh", "-c", "gadget-integration.test -test.v -integration -test.timeout 20m"]
```

Comment:
I am not sure about this one.
When I first wrote the integration tests, I added this flag, but after some discussion we decided to remove it.
I am old fashioned and I prefer "when nothing is printed, everything is OK".